lib/compilers/argument-parsers.ts

```diff
@@ -45,7 +45,7 @@ export class BaseParser {
     }
 
     static getExamplesRoot(): string {
-        return props.get('builtin', 'sourcePath', './examples/');
+        return props.get('builtin', 'sourcePath', './examples/') as string;
```

Comment:
i believe you can do `props.get<string>()`, right?

Reply:
Indeed! Fixed.